core/src/main/resources/lib/form/advanced.jelly

```diff
@@ -53,10 +52,10 @@ THE SOFTWARE.
 	      <input type="button" value="${attrs.title?:'%Advanced'}..." class="advanced-button advancedButton" /><!-- advancedButton is legacy -->
 	    </div>
             <j:new var="customizedFields" className="java.util.TreeSet"/>
-	    <table class="advancedBody"><tbody>
+	    <div class="advancedBody"><div class="tbody">
```

Comment:
similar to previous comment - using tbody seems to be confusing for future users.  its no longer a tbody

Reply:
it _might_ be possible to change this, I doubt it's targeted much, the tr one is a lot harder to change, as lots looks it up and there's some plugins we've changed to look at it